man/subscription-manager.8

```diff
@@ -83,59 +83,56 @@ must be passed as system arguments in a non-interactive session.
 4. auto-attach
 
 .IP
-5. remove
+5. release
```

Comment:
Please do not change numbers in this list. Just remove command "remove" from the list. Otherwise, it will cause lot of unnecessary conflicts (there are other PRs waiting for review removing something). Not only this list will be corrected during final review of this man page.

Reply:
Thank you for the feedback @jirihnidek. I had wondered if updating the list numbers to reflect the eliminated **_remove_** was a good idea or not. I can see the value of not doing it, as you point out.

This has been address in commit fcb99c338f77bae028d63570e2593ee6cbf04d89